### DISCUSSION TITLE
Hello to all guy's here

### DISCUSSION THREAD

**Discussion by Lakhan3011** [Cyfrin/foundry-full-course-cu#2148] [General]
I'm just started the course now. Is anyone also started, let's come together and have fun while learning and building together.. see you soon !!

**Comment by HosZxd** [comment]
let's do this

**Reply by EngrPips** [reply]
We can do it!

**Reply by Lakhan3011** [reply]
Let's connect @EngrPips

**Reply by EngrPips** [reply]
> Let's connect @EngrPips

Why not? I am engrpips on `x` and on discord

**Reply by Lakhan3011** [reply]
could you share the link of your profile?

**Reply by EngrPips** [reply]
https://x.com/EngrPips

**Reply by Lakhan3011** [reply]
Thanks

**Reply by Lakhan3011** [reply]
Not able to dm

**Comment by abubkar80** [comment]
hello guys

**Reply by EngrPips** [reply]
Hey, Friend. Welcome aboard.

**Reply by abubkar80** [reply]
how is the coding

**Reply by EngrPips** [reply]
Alhamdulillah

**Comment by Lakhan3011** [comment]
Good to see you, My Friend

**Reply by EngrPips** [reply]
It's all my pleasure buddy.

**Comment by Lakhan3011** [comment]
Where are you in the course,
I just deployed my first ever contract on zkSync testnet.

**Reply by EngrPips** [reply]
Finished the course except for the new Uniswap V2 content. I am redoing the `EVM & Formal Verification.` part again.

**Comment by Lakhan3011** [comment]
Ok , Great
Where are you based out from ?

**Reply by EngrPips** [reply]
Yeah, I am from Nigeria.

**Reply by abubkar80** [reply]
i am also from nigeria

**Reply by EngrPips** [reply]
Interesting. That's great to know.

**Comment by Lakhan3011** [comment]
Ok , Great.
What's your experience with the course so far?

**Reply by EngrPips** [reply]
It's great for me and teaches almost every basic I need to build on to be successful.

**Comment by Lakhan3011** [comment]
Glad to hear it!

**Reply by EngrPips** [reply]
Yes sir.

**Comment by Lakhan3011** [comment]
So far so good!

**Reply by EngrPips** [reply]
Indeed

**Comment by Lakhan3011** [comment]
Thanks for the tip!

**Reply by EngrPips** [reply]
You are welcome, Friend.

**Comment by Lakhan3011** [comment]
Thanks , Jai shree Ram

**Reply by EngrPips** [reply]
You are welcome.
